src/resources/views/crud/columns/range.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['text'] = $column['default'] ?? '-';
+    $column['progress_class'] = $column['progress_class'] ?? '';
```

Comment:
We need `progress-bar` class to properly align the element, not sure empty is a sensible default here.
![image](https://user-images.githubusercontent.com/7188159/198283695-afdc83c7-3173-49a0-9003-070a4f722f0c.png)
![image](https://user-images.githubusercontent.com/7188159/198283795-318c1671-0134-4f40-aca9-5bc8c5a7d7f0.png)

 We should make `progress-bar` a mandatory class and merge it with any others provided by developer, **not replace it or force the developer to provide it**.

Reply:
made `progress-bar` class default..

Reply:
Good catch @pxpm . Thanks @munjaldevelopment !